_data/apis.yml

```diff
@@ -208,7 +208,7 @@
   urls:
    - https://omniscien.com
    - https://omniscien.com/machine-translation/supported-languages/
-  languages: [ ar, be, bg, bn, ca, cz, da, de, el, en, es, et, fa, fi, fr, fr, ga, gu, he, hi, hr, hu, id, is, it, ja, km, ko, ku, lb, lt, lv, ms, mt, my, nl, 'no', pa, pl, pt, pt, ro, ru, si, sk, sl, so, sq, sr, sv, sw, ta, th, tk, tl, tr, uk, ur, vi, zh, zh ]
+  languages: [ ar, be, bg, bn, ca, cs, da, de, el, en, es, et, fa, fi, fr, fr-ca, ga, gu, he, hi, hr, hu, id, is, it, ja, km, ko, ku, lb, lt, lv, ms, mt, my, nl, 'no', pa, pl, pt-pt, pt-br, ro, ru, si, sk, sl, so, sq, sr, sv, sw, ta, th, tk, tl, tr, uk, ur, vi, zh-cn, zh-tw ]
```

Comment:
Odd that they did not have locales.  We should probably add a check for repeated language codes in the list.